### PR TITLE
Add device capabilities report

### PR DESCRIPTION
**Pull request by YouKnowBlom**

### CODE REVIEW

File: components/maincontroller.js [hunk 1]
```diff
@@ -195,6 +195,30 @@
             window.subtitleAppearance = data.subtitleAppearance;
         }
 
+        //Report device capabilities
```
Comment (cvium) [inline components/maincontroller.js]:
```suggestion
        // Report device capabilities
```

File: components/maincontroller.js [hunk 2]
```diff
@@ -195,6 +195,30 @@
...
+        if (!window.hasReportedCapabilities) {
+            //Kinda dirty solution but it'll do for now
```
Comment (cvium) [inline components/maincontroller.js]:
```suggestion
            // Kinda dirty solution but it'll do for now
```

Comment (cvium) [reply components/maincontroller.js]: Maybe explain why it is dirty and what would be better

Comment (YouKnowBlom) [reply components/maincontroller.js]: Good point, will do

**Comment by YouKnowBlom** [comment]
I went ahead and removed that comment cause I was under the impression that the device capabilities weren't persistent across server restarts. That would have in turn messed up HLS playback if the server was restarted while the CC is connected (since the device capabilities are only reported on initial connect).

**Review by cvium — APPROVED** [review]

**Pull request merged**